service/controller/v24/adapter/cpf_route_tables.go

```diff
@@ -2,109 +2,118 @@ package adapter
 
 import (
 	"context"
-	"fmt"
-	"strings"
 
-	"github.com/aws/aws-sdk-go/aws"
-	"github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/giantswarm/apiextensions/pkg/apis/provider/v1alpha1"
 	"github.com/giantswarm/microerror"
 
 	"github.com/giantswarm/aws-operator/service/controller/v24/controllercontext"
 	"github.com/giantswarm/aws-operator/service/controller/v24/encrypter"
-	"github.com/giantswarm/aws-operator/service/controller/v24/key"
+	"github.com/giantswarm/aws-operator/service/routetable"
```

Comment:
How come this isn't under a controller. Is it shared between controllers?

Reply:
I added godoc to the package. I thought about putting it to the controller but that would only mean a difference in version and this will be gone soon with VOO anyway. Also the route table names are configured via flags and never change at runtime and we do not version flags either.

Reply:
Cool that makes sense and thanks for adding the godoc.